tools/api-coverage/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "api-coverage"
+edition.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Probably should remove the default comment

Reply:
Fixed.